### PR TITLE
Convert UUIDs for suppressed features

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

File: build/lib/compendium-pack.ts [hunk 1]
```diff
@@ -331,6 +331,12 @@ class CompendiumPack {
         const convertOptions = { to: to === "ids" ? "id" : "name", map } as const;
 
         // Convert UUIDs found in places particular to certain item types
```
Comment (CarlosFdez) [inline build/lib/compendium-pack.ts:333]: Maybe the new stuff should have been put above this comment?

Comment (simonward) [reply build/lib/compendium-pack.ts:333]: Switched to use itemIsOfType() instead.

Comment (CarlosFdez) [reply build/lib/compendium-pack.ts:333]: Not actually resolved. The thing I'm concerned about is the proximity of the comment with what its commenting about. The way you have it, it looks like suppressedFeatures is the only thing this comment is for, but the if/else below is also relevant.

But I'm thinking more about it and we can just remove the comment entirely.

Comment (CarlosFdez) [inline build/lib/compendium-pack.ts:333]:
```suggestion
```

**Pull request merged**